Libplanet/Tx/ITxInvoice.cs

```diff
@@ -39,5 +40,20 @@ public interface ITxInvoice : IEquatable<ITxInvoice>
         /// A list of actions in this transaction.
         /// </summary>
         TxActionList Actions { get; }
+
+        /// <summary>
+        /// The maximum amount of <see cref="FungibleAssetValue"/> that the transaction author is
+        /// willing to pay for the transaction.
+        /// This can be <see langword="null"/> if align the handling of that transaction
+        /// with the <see cref="IBlockMetadata.ProtocolVersion"/>.
```

Comment:
Wait, does this mean metering gas is dependent on `IBlockMetadata.ProtocolVersion`? 🤔

Reply:
amended. https://github.com/planetarium/libplanet/pull/3201/commits/2ad90d87437c3fd5d266da40052ef27a41fe3426